### PR TITLE
added force-reset in the init script

### PR DESCRIPTION
**Pull request by mabhi**
Signed-off-by: mabhi <abhijit.mukherjee@infracloud.io>

### What does this PR change?

- Set the `ForceReset` flag to the new admin user creation during deployment

### Does the PR depend on any other PRs or Issues? If yes, please list them.

- #115

### Checklist

I confirm, that I have...

- [ ] Read and followed the contributing guide in `CONTRIBUTING.md`
- [ ] Added tests for this PR
- [ ] Formatted the code using `go fmt` (if applicable)
- [ ] Updated [documentation on the Paralus docs site](https://github.com/paralus/website/blob/main/docs) (if applicable)
- [ ] Updated `CHANGELOG.md`

### CODE REVIEW

**Comment by mabhi** [comment]
@akshay196 please check

**Comment by mabhi** [comment]
- [ ] instead of printing the recovery url we print the generated password
- [ ] check with pctl (Add user)
- [ ] Adding / Creating new user should now respond with password generated instead of recovery url and reset `forceReset` flag so users are forced to set their own password

File: pkg/reconcile/cluster_reconciler.go [hunk 1]
```diff
@@ -66,18 +75,93 @@ func canReconcileClusterBootstrapAgent(c *infrav3.Cluster) bool {
 	}
 }
 
+// DeleteForCluster delete bootstrap agent
+func (r *clusterReconciler) deleteBootstrapAgentForCluster(ctx context.Context, cluster *infrav3.Cluster) error {
```
Comment (niravparikh05) [inline pkg/reconcile/cluster_reconciler.go:79]: why are we seeing these changes in this pr? they are already merged to main and released right ?

**Review by niravparikh05 — CHANGES_REQUESTED** [review]
please rebase with main, ensure current pr has only the changes with respect to the feature.

**Comment by akshay196** [comment]
@mabhi Please move https://github.com/paralus/paralus/pull/123/commits/e5245f140ce26fecf4d864719322d3085bebf921 to [autogenerate-password-and-forcereset](https://github.com/paralus/paralus/pull/124) branch as it combines all Paralus PRs related to issue #92

File: pkg/reconcile/cluster_reconciler.go (hunk 1 above)
Comment (mabhi) [reply pkg/reconcile/cluster_reconciler.go:79]: Yes you are correct. I understand this should not be part of this PR, but it came during rebase with the main. Now a new branch has come into place so we are merging only the required changes into it.

**Comment by mabhi** [comment]
@akshay196, Done. [commit](https://github.com/paralus/paralus/pull/124/commits/47f2d1e392cb054ba2ab673c57909bb152f39669)

**Comment by akshay196** [comment]
Closing as commits are moved to another PR https://github.com/paralus/paralus/pull/124

**Pull request closed without merging**